app/forms/webauthn_setup_form.rb

```diff
@@ -39,12 +40,14 @@ def self.domain_name
   private
 
   attr_reader :success
-  attr_accessor :user, :challenge, :attestation_object, :client_data_json, :name
+  attr_accessor :user, :challenge, :attestation_object, :client_data_json,
+                :name, :platform_authenticator
 
   def consume_parameters(params)
     @attestation_object = params[:attestation_object]
     @client_data_json = params[:client_data_json]
     @name = params[:name]
+    @platform_authenticator = (params[:platform_authenticator] == true)
```

Comment:
`params` from the URL comes back with string values, I think we need some casting here:

```suggestion
    @platform_authenticator = (params[:platform_authenticator].to_s == 'true')
```